CLOUDP-261756: Repository setup for Postman Project (#74)
mongodb/openapi: tools/postman/README.md

# Postman Collections from OpenAPI

> Folder contains scripts and other files required to generate a Postman collections from the OpenAPI Specification

Reviewer: ```suggestion
This folder contains the code required to generate the MongoDB Atlas Postman collections from the MongoDB OpenAPI Specification
```

    C --> D[Convert to\nPostman Collection]
    D --> E[Apply JSON\ntransformations]
    E --> F[Upload Collection\nto Postman]
    F --> G[Wait 24 hours]
    G --> A[Fetch the OpenAPI\nSpecification]

    style A width:160px,height:60px,text-align:center;
    style B width:160px,height:60px,text-align:center;
    style C width:160px,height:60px,text-align:center;
    style D width:160px,height:60px,text-align:center;
    style E width:160px,height:60px,text-align:center;
    style F width:160px,height:60px,text-align:center;
    style G width:160px,height:60px,text-align:center;
```

1. **Fetch the OpenAPI Specification**: Periodically Fetch for the Changes in the Public OpenAPI file.

2. **Has the OpenAPI file changed**: Checking for that the OpenAPI file has changed stops redundant updates to the
   Collection.

3. **Apply OpenAPI transformations**: Apply transformations such as removing regex to prepare openapi file for
   conversion.

4. **Convert to Postman Collection**: Use openapi-to-postmanv2 to convert to a Postman Collection.

5. **Apply JSON transformations**: Apply transformations such as changing title, adding fields, and preparing Collection
   for upload.

> 6. **Upload Collection to Postman**: Use the Postman API to upload the Collection to Postman.

Reviewer: could you remove the file `.keep` files from the PR?

Reviewer 2: Idea is to have gitignore for the entire folder that will keep .keep files forever.
